bad_rows.txt

```diff
@@ -0,0 +1,782 @@
+House Committees
```

Comment:
Do we want this file under version control?

Reply:
No, adding to .gitignore